pom.xml

```diff
@@ -242,21 +253,24 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
-                    <compilerId>javac-with-errorprone</compilerId>
+                    <!-- compilerId>javac-with-errorprone</compilerId -->
                     <forceJavacCompilerUse>true</forceJavacCompilerUse>
                     <showDeprecation>true</showDeprecation>
+                    <source>${source.jdk.version}</source>
+                    <target>${target.jdk.version}</target>
+                    <compilerArgument>${compilerArgument}</compilerArgument>
```

Comment:
I don't understand the purpose of these changes. Can you elaborate?

Reply:
Hi @bernd, I commented it because I remember compiling with that entry enabled I got into a bunch of errors. Tried to google a bit and got answers that says we needed to use a more recent version of the library but I remember I continued to not being able to compile. Let me review the things and I will be back to you with fresh news because I decided to submit the PR one month later I applied these changes and frankly I don't remember. Talk to you soon.